modules/web-console/frontend/app/vendor.js

```diff
@@ -52,12 +52,9 @@ import 'file-saver';
 import 'jszip';
 import 'nvd3';
 import 'lodash';
-import 'angular-ui-carousel';
 
 import 'angular-gridster/dist/angular-gridster.min.css';
 import 'angular-tree-control/css/tree-control-attribute.css';
 import 'angular-tree-control/css/tree-control.css';
 import 'angular-ui-grid/ui-grid.css';
-import 'angular-motion/dist/angular-motion.css';
 import 'nvd3/build/nv.d3.css';
-import 'angular-ui-carousel/dist/ui-carousel.css';
```

Comment:
@AlexDel page-landing uses ui-carousel. Either restore the dependency or remove ui-carousel from landing (maybe use ignite logo instead?).